config.yaml

```
@@ -221,6 +221,8 @@ settings:
   clustername: not-configured
   # Set true to enable kubectl commands execution
   allowkubectl: false
+  # set Namespace to execute botkube kubectl commands by default
+  kubectlNamespace: default
```

Should this be `defaultNamespace` instead of `kubectlNamespace` ? Kubectl namespace is bit misleading considering how do we pass other namespace.

---

considering that `defaultNamespace` would be misleading, as we have `namespaces` under resources sections as well earlier, came up with `kubectlNamespace`. 
Any Alternatives to both of them? like `defaultKubeCmdNs`

---

resolved this issue.